File: Array&String/String/CPP/greatest_english_letter_in_upper_and_lower_case.cpp
#include <string>
using namespace std;

class Solution {
    public:
        string greatestLetter(string s) {
            for (int i = 25; i > -1; i--) {
                char up = 65 + i,
                     low = 97 + i;
                if (s.find(up) != string::npos && s.find(low) != string::npos) return string(1, up);
            }
            return "";
        }
    };

File: Array&String/String/JS/greatest_english_letter_in_upper_and_lower_case.js
/**
 * @param {string} s
 * @return {string}
 */
var greatestLetter = function(s) {
    for (let i = 25; i > -1; i--) {
        const up = String.fromCharCode(65 + i),
              low = String.fromCharCode(97 + i);
        if (s.includes(up) && s.includes(low)) return up;
    }
    return "";
};